### PR TITLE
added packages/setcore/setcore.1.0.0

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

**Comment by UnixJunkie** [comment]
I want to check that it works with all OCaml versions.

**Comment by UnixJunkie** [comment]
All CI tests are green.

File: packages/setcore/setcore.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+dev-repo: "git+https://github.com/UnixJunkie/setcore.git"
+bug-reports: "https://github.com/UnixJunkie/setcore/issues"
+build: [
+  ["autoconf"]
+  ["autoheader"]
+  ["./configure"]
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "conf-autoconf"
```
Comment (kit-ty-kate) [inline packages/setcore/setcore.1.0.0/opam]:
```suggestion
  "conf-autoconf" {build}
```

File: packages/setcore/setcore.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "ocaml"
+  "ocamlfind"
```
Comment (kit-ty-kate) [inline packages/setcore/setcore.1.0.0/opam]: are you sure you need ocamlfind? How is it required in the build process, I can only see it in configure but no ocaml libraries are needed

Comment (kit-ty-kate) [reply packages/setcore/setcore.1.0.0/opam]: At the very least if it's really needed can you tag it as a `build` dependency too?

Comment (UnixJunkie) [reply packages/setcore/setcore.1.0.0/opam]: Yes, ocamlfind is just some left-over piece in the configure things.
I will remove it and retag the software.

**Comment by UnixJunkie** [comment]
I removed the ocamlfind dependency and issued a new tag for the software.

**Comment by UnixJunkie** [comment]
All CI tests are green.

**Comment by mseri** [comment]
Thanks

**Pull request merged**